content/documentation/device-examples/components/LCDDisplay.md

```diff
@@ -8,7 +8,7 @@ The [LCD Display](https://github.com/Pi4J/pi4j-example-components/tree/main/src/
 It is used to show Numbers, Text and Symbols on a small Display.
 The Class supports only LCD Displays with the PCF8574T I2C Backpack. Supported display-dimensions are 40x2, 20x4, 20x2, 16x2, 16x1.
 
-{{% notice note %}}
+{{% notice warning %}}
```

Comment:
why warning, note fits better in this situation

Reply:
corrected